src/cuter_codeserver.erl

```diff
@@ -209,7 +211,7 @@ handle_info(_Msg, State) ->
                .
 handle_call({load, M}, _From, State) ->
   {reply, try_load(M, State), State};
-handle_call({get_spec, {M, F, A}=MFA}, _From, State) ->
+handle_call({get_spec, {M, F, A}=MFA}, _From, State=#st{normalizeTypes = NormalizeTypes}) ->
```

Comment:
The placement here is inconsistent.  On the first the variable is on the right and the second one is on the left.  (If you want my opinion, placing the variable on the right is better.)